src/compiler/Error.elm

```diff
@@ -60,6 +60,8 @@ type ParseContext
     | InLambda
     | InList
     | InUnit
+    | InTuple2
```

Comment:
Let's name the normal tuple `Tuple` and the triple `Tuple3` :) No need for the `2`.

Reply:
I've been considering proposing using the notation the original compiler uses and updating all the other types, what do you think?

Reply:
@halfzebra I might be a bit confused now - you mean like this? https://github.com/elm/compiler/blob/master/compiler/src/AST/Source.hs#L66

Reply:
I was thinking about the notation used for [JS constructors](https://github.com/elm/compiler/blob/33f153eea8222b481fcf2f6d42c83b1230d00d06/compiler/src/Generate/JavaScript/Expression.hs#L171).

I'll make it consistent first 🙂